lib/matplotlib/backends/backend_qt5.py

```diff
@@ -442,6 +464,52 @@ def closeEvent(self, event):
         QtWidgets.QMainWindow.closeEvent(self, event)
 
 
+class Window(WindowBase, MainWindow):
+    def __init__(self, title):
+        WindowBase.__init__(self, title)
```

Comment:
This is going to be subtle to get right because some versions of pyqt do not user `super` and some do.  This will probably need some conditional logic based on pyqt version.